Gemfile

```diff
@@ -9,11 +9,12 @@ gem "rails", "~> 5.0.4"
 gem "sqlite3"
 gem "puma", "~> 3.0"
 gem "sass-rails", "~> 5.0"
-gem "uglifier", ">= 1.3.0"
+gem "uglifier", "3.2.0"
 gem "coffee-rails", "~> 4.2"
 gem "jquery-rails"
 gem "turbolinks", "~> 5"
 gem "jbuilder", "~> 2.5"
+gem "therubyracer"
```

Comment:
Fix lỗi: therubyracer (V8) runtime is not available on this system...
Dẫu biết không nên dùng nhưng do cài Nodejs mà vẫn không fix được..